docs/docs/guides/integrate/login-ui/device-auth.mdx

```diff
@@ -0,0 +1,167 @@
+---
+title: Support for the Device Authorization Grant in a Custom Login UI
+sidebar_label: Device Authorization
+---
+
+In case one of your applications requires the OAuth2 Device Authorization Grant this guide will show you how to implement
+this in your application as well as the custom login UI.
```

Comment:
Perhaps link to the device specific guide as well?

```suggestion
In case one of your applications requires the [OAuth2 Device Authorization Grant](/docs/guides/integrate/login/oidc/device-authorization) this guide will show you how to implement
```

Reply:
Hmm this got connected to the wrong line. Gimme a sec to resend the suggestion.